kibana/nodescl-node

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# Copyright 2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+COMMAND="node $@"
+scl enable rh-nodejs6 "${COMMAND}"
```

Comment:
I followed the pattern https://www.softwarecollections.org/en/docs/guide/#sect-Packaging_Wrappers_for_Software_Collections but if it works the way you describe I'm ok with that